src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -1075,6 +1080,8 @@ func (i *fetchTaggedResultsIter) Close(err error) {
 	i.totalMetrics.ByRange.Record(queryRange, totalFetchTime)
 	i.totalMetrics.ByDocs.Record(i.totalDocsCount, totalFetchTime)
 
+	i.totalBlocks.Update(float64(i.totalBlocksCount))
```

Comment:
Is a gauge what we want here? Won't we miss a lot of the potential fluctuation based on just what the val happens to be on each scrape?

Reply:
Ugh, I forgot about the scrape interval. What's the standard way of getting metrics per request?